recipes/mongodb_org_repo.rb

```diff
@@ -38,7 +38,7 @@
 when 'rhel', 'fedora'
   yum_repository 'mongodb' do
     description 'mongodb RPM Repository'
-    baseurl "http://downloads-distro.mongodb.org/repo/redhat/os/#{node['kernel']['machine']  =~ /x86_64/ ? 'x86_64' : 'i686'}"
+    baseurl "#{node['mongodb']['repo']}/#{node['kernel']['machine']  =~ /x86_64/ ? 'x86_64' : 'i686'}"
```

Comment:
What are your thoughts on using strings here?  Would it be possible to to keep symbols to be more consistent with previous references?